airbyte-integrations/connectors/source-relational-db/src/main/java/io/airbyte/integrations/source/relationaldb/DbSourceDiscoverUtil.java

```diff
@@ -52,11 +53,19 @@ public static <DataType> void logSourceSchemaChange(final Map<String, TableInfo<
           .distinct()
```

Comment:
I know it's not related to your change - but could you update the comment for `logSourceSchemaChange` to say : "This method logs schema drift between source table and the catalog. This can happen if (i) underlying table schema changed between syncs or (ii) The source connector's mapping of data types to Airbyte types changed between runs".